R/Factbase.R

```diff
@@ -383,15 +385,18 @@ UpdateFactbasePenetrationFormula <- function(metric_name, token, numerator, deno
     if (test_return_json)
         return(body)
     
-    url <- paste0("https://factbase.azurewebsites.net/formula?metric=", URLencode(metric_name), "&definition=", URLencode(definition))
-    if (!is.null(hyperlink))
-        url <- paste0(url, '&hyperlink=', hyperlink)
+    url <- paste0(
+        "https://factbase.azurewebsites.net/formula?metric=",
+        URLencode(metric_name),
+        "&definition=", URLencode(definition),
+        "&definition=", URLencode(hyperlink),
+        "&definition=", URLencode(owner))
```

Comment:
Should the two `&definition=` strings be `&hyperlink=` and `&owner=` here?

Reply:
Good catch.  Thanks, @jrwishart